### PR TITLE
ref(register): Add password strength component

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This component replaces the 1st party pwStrength package that we have with a react component that performs the same functionality.

Importantly, this component is NOT loaded anywhere within the main react app, and instead is exposed as a asynchronously, code-split, module that can be loaded through the global SentryApp variable.

A shim is provided through the exported `attachTo` function to allow the password strength component to be rendered on non-react pages.

Supersedes: https://github.com/getsentry/sentry/pull/11072

pw

image

### CODE REVIEW

**Review by markstory — APPROVED** [review]
Nice work!

**Comment by EvanPurkhiser** [comment]
Going to wait for @billyvg to take a quick peek as well before merging.

**Review by billyvg — APPROVED** [review]
Damn, this is much cleaner, good stuff!

File: src/sentry/static/sentry/app/components/passwordStrength.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+     */
+    colors: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * The password value.
+     */
+    value: PropTypes.string,
+  };
+
+  static defaultProps = {
+    labels: ['Very Weak', 'Very Weak', 'Weak', 'Strong', 'Very Strong'],
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/passwordStrength.jsx]: Should probably have translated these

**Pull request merged**